README.md

```diff
@@ -97,23 +98,45 @@ polymer-bundler as a library has two exported function.
 
 `.bundle()` takes a `BundleManifest` and returns a promise to a `DocumentCollection` of the generated bundles.
 
-Example:
+A simple example:
+```js
+var parse5 = require('parse5');
+var bundler = new require('polymer-bundler')();
+bundler.generateManifest(['my-app.html']).then((manifest) => {
+  bundle(manifest).then((bundles) => {
+    console.log('<!-- BUNDLED VERSION OF my-app.html: -->');
+    console.log(parse5.serialize(bundles.get('my-app.html').ast));
+  });
+});
+```
+
+An example, using options:
```

Comment:
Consider:

`An example with a customized sharding strategy and output layout:`